crc/aarch64/crc32_gzip_refl_pmull.h

```diff
@@ -47,14 +47,14 @@
 .equ	br_high_b2, 0x1
 
 	.text
-#ifndef __MACH__
+#ifndef __APPLE__
 	.section	.rodata
 #else
   .section	__DATA,data
```

Comment:
miss this message

Reply:
umm, let me take a look at this later. Without careful I might mention something wrong.

Reply:
__TEXT,__const is appropriate as mentioned above